src/less/general.less

```diff
@@ -46,7 +46,6 @@
 div.gradient-button, .gradient-button {
 	cursor: pointer;
 	text-decoration: none;
-	user-select: none;
```

Comment:
Would it make sense to keep these two `user-select: none;` statements and just not use grunt-autoprefixer? Then we could keep the button's text disabled like how it was, and simplify the Gruntfile. According to Duck AI, modern browsers support user-select, so maybe removing grunt-autoprefixer won't break any of the Grade A browsers.

<img width="612" alt="image" src="https://github.com/user-attachments/assets/5e8be2b2-ee79-47d6-9a06-ac7e47c61572" />

Reply:
I only glanced at MDN earlier where it says Limited Availablity. But indeed turns out it's supported everywhere except for Safari which needs a -webkit- prefix. So we could just add the webkit prefix manually for this rule. Done.